### ISSUE TITLE
don't show a "point" for an object that is not a node

### ISSUE THREAD

**Issue by TimMagee** [facebook/Rapid#958]
### Expected Behavior
The label for an object (such as a gas station) marked with a "way" (an area) is visually distinct from one marked as "node". 

### Actual Behavior
When an object is marked as an area, the label shows a "pointed bubble" just like for a node. This is confusing.

### Behavior in iD
iD just shows the label for the object with the icon for the object type. The icon is notably not in a "pointed bubble" as for a node.

### Suggested behavior
Either show the icon just like in iD or place it in a square with dots at the corners to visually show that the object is an area as opposed to a node.

### Version

2.0.3

### What browser are you seeing the problem on? What version are you running?

Firefox v102.0

### The OS you're using

linux

### Steps to reproduce

1. Open Rapid to anywhere that has objects tagged on both "areas" and "nodes".
2. Notice that the icons beside the labels are not visually distinct and are instead confusingly the same.

### The browser URL at the time you encountered the bug

https://rapideditor.org/edit#background=Bing&datasets=fbRoads,298703363f16457e9332c431e344470b&disable_features=boundaries&map=19.92/37.42719/-79.15300

### The auto-detected useragent string for your browser (leave blank if you're manually filling this form out)

Mozilla/5.0 (X11; Linux x86_64; rv:102.0) Gecko/20100101 Firefox/102.0

**Comment by bhousel** [comment]
I agree we can improve this somehow - After using this for the past few months I do get confused sometimes.

#534 has the context on why we added the virtual point, and the sorts of situations where it makes the rendering less confusing.  I'm open to adjusting the design of it somehow so that it looks different from a true point.
